pom.xml

```diff
@@ -94,13 +94,56 @@
     </issueManagement>
 
     <properties>
+        <java.version>11</java.version>
```

Comment:
all these pom.xml modifications must be in the ecocode-rules-specifications pom.x instead and the current parent pom.xml

all other plugins work with java 17. why do you put java 11 ?

Reply:
POM changes have been moved to ecocode-rules-specifications.

Some of the configuration, including java 11 is coming from
https://github.com/green-code-initiative/ecoCode-java/blob/main/pom.xml

Java version could be changed.